test/Microsoft.AspNetCore.Razor.Language.Test/IntegrationTests/TestTagHelperDescriptors.cs

```diff
@@ -158,7 +162,7 @@ public static IEnumerable<TagHelperDescriptor> EnumTagHelperDescriptors
                                 .Name("catch-all")
                                 .PropertyName("CatchAll")
                                 .AsEnum()
-                                .TypeName(typeof(MyEnum).FullName),
+                                .TypeName(GetFullName(typeof(MyEnum))),
```

Comment:
Frankly, this whole approach is wrong. We gain no value from doing a `.Equals()` on the descriptors. It would be much better to have the tests validate their expectations explicitly.